AppNavigator.js

```diff
@@ -5,10 +5,11 @@ import LoginScreen from "./components/LoginScreen";
 import ChatScreen from "./components/ChatScreen"
 import RegisterScreen from "./components/RegisterScreen";
 
+
 const AppNavigation = createStackNavigator({
-    Chat: ChatScreen,
-    Login: LoginScreen,
-    Register: RegisterScreen
+    Chat: { screen: ChatScreen, navigationOptions: { title: 'Chat', headerLeft: null }},
```

Comment:
what is the purpose of `headerLeft` config prop?

Reply:
"headerLeft: null" used to remove "BACK" button (screenshot) from the page, because for these pages we don’t need to go back.
![Screen Shot 2020-01-02 at 11 10 59 AM](https://user-images.githubusercontent.com/23140788/71659525-a1003380-2d50-11ea-9118-0f6ddaac3068.png)